docs/src/tutorials/bseries_basics.ipynb

```diff
@@ -0,0 +1,653 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we use `bseries.jl` to investigate error expansions for RK methods applied to specific problems."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the packages we will use.  These must first be installed using: import Pkg; Pkg.add(\"package_name\")\n",
+    "using BSeries\n",
+    "using Latexify\n",
+    "using RootedTrees\n",
+    "using Symbolics\n",
+    "import SymPy; sp=SymPy;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# B-Series for a generic ODE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we specify the Butcher coefficients of the RK method.  This can include symbolic expressions and parameterized families of methods.  Here is a generic 2-stage, 2nd-order method."
```

Comment:
When convertig this to Markdown, it would be nice to restrict the line length a bit to make it easier to review changes/code via GitHub.